Labs/module-1_labs/03_mysql/lab_sql-extras/olist_data/exercises/lab-1-olist.md

```diff
@@ -0,0 +1,153 @@
+![IronHack Logo](https://s3-eu-west-1.amazonaws.com/ih-materials/uploads/upload_d5c5793015fec3be28a63c4fa3dd4d55.png)
+
+# Lab | SQL
+
+## Introduction
+
+Run your first SQL commands! In this lab, we will practice selecting and projecting data. You can finish all questions with only those clauses:
+- `SELECT`
+- `SELECT DISTINCT`
+- `COUNT`
+- `FROM`
+- `WHERE`
+- `ORDER BY`
+- `GROUP BY`
+- `SUM`
```

Comment:
`SUM()`, maybe explicitly group both the `COUNT()` and the `SUM()` function separately.